hazelcast/src/main/java/com/hazelcast/concurrent/lock/operations/LocalLockCleanupOperation.java

```diff
@@ -35,6 +35,14 @@
 
     private final String uuid;
 
+    /**
+     * This constructor should not be used to obtain an instance of this class; it exists to fulfill IdentifiedDataSerializable
+     * coding conventions.
+     */
+    public LocalLockCleanupOperation() {
+        uuid = "";
```

Comment:
You're right, what we lose with throwing `UnsupportedOperationException` in the constructor is the opportunity to verify that the factoryId-id combination is either throwing `UnsupportedOperationException` or unique. Maybe the utility of failing fast and hard instead of relying on reading the javadoc in a not-supposed-to-be-used constructor is greater than being able to instantiate every `IdentifiedDataSerializable` class. @tombujok wdyt?